src/EnergyPlus/HeatingCoils.cc

```diff
@@ -3320,6 +3344,73 @@ namespace HeatingCoils {
 
 	}
 
+	void
+	SetHeatingCoilData(
+		int const CoilNum, // Number of electric or gas heating Coil
+		bool & ErrorsFound, // Set to true if certain errors found
+		Optional_bool DesiccantRegenerationCoil, // Flag that this coil is used as regeneration air heating coil
+		Optional_int DesiccantDehumIndex // Index for the desiccant dehum system where this caoil is used 
+		) {
+
+		// FUNCTION INFORMATION:
+		//       AUTHOR         Bereket Nigusse
+		//       DATE WRITTEN   February 2016
+		//       MODIFIED       na
+		//       RE-ENGINEERED  na
+
+		// PURPOSE OF THIS FUNCTION:
+		// This function sets data to Heating Coil using the coil index and arguments passed
+		// If incorrect coil index is passed, ErrorsFound is returned as true
+
+		// METHODOLOGY EMPLOYED:
```

Comment:
@Nigusse If this function stays, please delete the unused boiler-plate comments here and below.